### PR TITLE
Deindent to unbreak sphinx autodoc

### PR DESCRIPTION
**Pull request by alex-ozdemir**

### CODE REVIEW

File: cvc5_z3py_compat/z3.py [hunk 1]
```diff
@@ -7399,8 +7399,8 @@ def sort(self):
 
 def TupleSort(name, sorts, ctx=None):
     """Create a named tuple sort base on a set of underlying sorts
-    Example:
-        >>> pair, mk_pair, (first, second) = TupleSort("pair", [IntSort(), BoolSort()])
+
```
Comment (nafur) [inline cvc5_z3py_compat/z3.py:7402]: It may be good to document what this method returns...

File: cvc5_z3py_compat/z3.py [hunk 2]
```diff
@@ -7411,8 +7411,8 @@ def TupleSort(name, sorts, ctx=None):
 
 def DisjointSum(name, sorts, ctx=None):
     """Create a named tagged union sort base on a set of underlying sorts
-    Example:
-        >>> sum, ((inject0, extract0), (inject1, extract1)) = DisjointSum("+", [IntSort(), BoolSort()])
+
```
Comment (nafur) [inline cvc5_z3py_compat/z3.py:7414]: Same here.

File: cvc5_z3py_compat/z3.py (hunk 1 above)
Comment (alex-ozdemir) [reply cvc5_z3py_compat/z3.py:7402]: Agreed.

File: cvc5_z3py_compat/z3.py (hunk 2 above)
Comment (alex-ozdemir) [reply cvc5_z3py_compat/z3.py:7423]: Agreed.

File: cvc5_z3py_compat/z3.py [hunk 3]
```diff
@@ -7400,7 +7400,16 @@ def sort(self):
...
 
+    Returns the tuple datatype, its constructor, and a list of accessors, in order.
+
     >>> pair, mk_pair, (first, second) = TupleSort("pair", [IntSort(), BoolSort()])
+    >>> b = Bool('b')
+    >>> i = Int('i')
+    >>> p = mk_pair(i, b)
+    >>> p
+    pair(i, b)
+    >>> solve([b != second(mk_pair(i, b))])
```
Comment (nafur) [inline cvc5_z3py_compat/z3.py:7411]:
```suggestion
    >>> solve([b != second(p)])
```

**Pull request merged**